config/application.rb

```diff
@@ -24,7 +24,7 @@
 module App
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 7.0
+    config.load_defaults 7.1
 
     config.active_support.cache_format_version = 7.1
```

Comment:
Can you also remove this line in this commit? Presumably `load_defaults 7.1` includes setting `cache_format_version = 7.1`.

Reply:
Fixed in bb505e4